CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -69,10 +69,16 @@ class HomeViewController: UIViewController {
 
     private func getUserMatching() {
         guard let netId = UserDefaults.standard.string(forKey: Constants.UserDefaults.userNetId) else { return }
-        NetworkManager.shared.getUser(netId: netId).chained { (response: Response<User>) -> Future<Response<Matching?>> in
+        NetworkManager.shared.getUser(netId: netId).chained { [weak self] (response: Response<User>) -> Future<Response<Matching?>> in
+            guard let self = self else { return Promise<Response<Matching?>>(error: NetworkingError.failed("Self invalid")) }
             if response.success {
                 let user = response.data
                 self.user = user
+                DispatchQueue.main.async {
+                    if let profilePictureURL = URL(string: user.profilePictureURL) {
+                        self.profileButton.kf.setImage(with: profilePictureURL, for: .normal)
+                    }
```

Comment:
is there a default image if this is unsuccessful?

Reply:
right now no, it will just be a gray green circle. but I think as long as we use google log in, there will always be an image.